libr/core/cmd_anal.c

```diff
@@ -729,6 +729,16 @@ static const char *help_msg_ax[] = {
 	NULL
 };
 
+static const char *help_msg_axt[]= {
+	"Usage:", "axt[?gq*]", "find data/code references to this address",
+	"axtj", " [addr]", "find data/code references to this address and print in json format",
+	"axtg", " [addr]", "display commands to generate graphs according to the xrefs",
+	"axtq", " [addr]", "find and list the data/code references in quiet mode",
+	"axt*", " [addr]", "same as axt, but prints as r2 commands",
+	"axt?", " ","Show this help",
```

Comment:
why `" "` and not `""` ? anyway, theres no need to add this line, the `?` is already in the usage line